pykern/pkcollections.py

```diff
@@ -51,6 +51,8 @@ def __copy__(self):
         return self.__class__(self)
 
     def __deepcopy__(self, memo):
+        # use self.copy() instead of self.__class__(), because subclasses
+        # of PKDict may have constructors with different numbers of parameters
```

Comment:
I think this should be a docstring. It should explain its arguments even though they are "standard". Also explain the coupling with `__copy__` and document it. A bit more filler text in the description.

Reply:
73403f857b02d89c940fc30f8c029f44e9621306